cli/packages/vim-lightline/lib/index.js

```diff
@@ -41,21 +85,21 @@ const render = (colors) => {
   }
 
   let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
-  let s:p.normal.left = [ [ s:shade1, s:accent5 ], [ s:shade7, s:shade2 ] ]
-  let s:p.normal.right = [ [ s:shade1, s:shade4 ], [ s:shade5, s:shade2 ] ]
-  let s:p.inactive.right = [ [ s:shade1, s:shade3 ], [ s:shade3, s:shade1 ] ]
-  let s:p.inactive.left =  [ [ s:shade4, s:shade1 ], [ s:shade3, s:shade0 ] ]
-  let s:p.insert.left = [ [ s:shade1, s:accent3 ], [ s:shade7, s:shade2 ] ]
-  let s:p.replace.left = [ [ s:shade1, s:accent1 ], [ s:shade7, s:shade2 ] ]
-  let s:p.visual.left = [ [ s:shade1, s:accent6 ], [ s:shade7, s:shade2 ] ]
-  let s:p.normal.middle = [ [ s:shade5, s:shade1 ] ]
-  let s:p.inactive.middle = [ [ s:shade4, s:shade1 ] ]
-  let s:p.tabline.left = [ [ s:shade6, s:shade2 ] ]
-  let s:p.tabline.tabsel = [ [ s:shade6, s:shade0 ] ]
-  let s:p.tabline.middle = [ [ s:shade2, s:shade4 ] ]
+  let s:p.normal.left = [ [ s:guishade1, s:guiaccent5, s:shade1, s:accent5 ], [ s:guishade7, s:guishade2, s:shade7, s:shade2 ] ]
+  let s:p.normal.right = [ [ s:guishade1, s:guishade4, s:shade1, s:shade4 ], [ s:shade5, s:shade2 ] ]
+  let s:p.inactive.right = [ [ s:guishade1, s:guishade3, s:shade1, s:shade3 ], [ s:shade3, s:shade1 ] ]
+  let s:p.inactive.left =  [ [ s:guishade4, s:guishade1, s:shade4, s:shade1 ], [ s:shade3, s:shade0 ] ]
+  let s:p.insert.left = [ [ s:guishade1, s:guiaccent3, s:shade1, s:accent3 ], [ s:shade7, s:shade2 ] ]
+  let s:p.replace.left = [ [ s:guishade1, s:guiaccent1, s:shade1, s:accent1 ], [ s:shade7, s:shade2 ] ]
+  let s:p.visual.left = [ [ s:guishade1, s:guiaccent6, s:shade1, s:accent6 ], [ s:shade7, s:shade2 ] ]
```

Comment:
Do we need to add the `gui` ones to the second list for these lines as well (as you have done on line 88), or no?

Reply:
Well, `:let g:lightline#colorscheme#default#palette` has them, so I guess we do need them to comply with lightline.